week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -13,8 +13,17 @@
  *      .....
  * </div>
  */
-function exerciseOne(arrayOfPeople) {
+unction exerciseOne(arrayOfPeople) {
   let content = document.querySelector("#content");
+ 
+  var h1Tag = document.createElement("h1");
+  var h2tag = document.createElement("h2");
+  content.appendChild(h1Tag);
+  content.appendChild(h2tag);
+ for(i=0; i<arrayOfPeople.length; i++){
```

Comment:
Were you able to test this code? Try to think about what will happen in this loop. 

Are you adding a h1 and a h2 for each person in the array? Or are you replacing the contents of h1 and h2 with the next array person each time?